pkgs/development/python-modules/python-efl/default.nix

```diff
@@ -48,6 +48,8 @@ buildPythonPackage rec {
     platforms = platforms.linux;
     license = with licenses; [ gpl3 lgpl3 ];
     maintainers = with maintainers; [ matejc ftrvxmtrx ] ++ teams.enlightenment.members;
-    broken = true;
+    # The generated files in the tarball aren't compatible with python 3.11
+    # See https://sourceforge.net/p/enlightenment/mailman/message/37794291/
+    broken = python.pythonAtLeast "3.11";
```

Comment:
This should have been disabled using disable

Reply:
Any documentation or example on how to do it with disable?

Reply:
```
disable = python at least "3.11";
```

Just ripgrep the code and you'll find some examples :)